src/it/default-fork-windows/pom.xml

```diff
@@ -30,14 +30,6 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   </properties>
 
-  <dependencies>
-    <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>3.8.2</version>
-    </dependency>
-  </dependencies>
-
```

Comment:
With above proposal to keep some dependency to a testing framework (any of them), it could be:

```xml
    <dependency>
      <groupId>org.junit.jupiter</groupId>
      <artifactId>junit-jupiter-api</artifactId>
      <version>5.14.0</version>
      <scope>test</scope>
    </dependency>
```

Or any other version of JUnit, or any other testing framework. The important thing is to have something with the `test` scope.

Note: I'm not sure why `<scope>test</scope>` was not present in the previous version of this test.